src/applications/personalization/profile/components/direct-deposit/BankInfoCNP.jsx

```diff
@@ -264,119 +256,80 @@ export const BankInfoCNP = ({
     return data;
   };
 
-  const mfaHandler = isAuthenticatedWithSSO => {
-    recordEvent({ event: 'multifactor-link-clicked' });
-    mfa(isAuthenticatedWithSSO ? 'v1' : 'v0');
-  };
-
   // Render nothing if the user is not LOA3.
   // This entire component should never be rendered in that case; this just
   // serves as another layer of protection.
   if (!isLOA3) {
     return null;
   }
 
-  if (showSetup2FactorAuthentication) {
```

Comment:
These `BankInfo` components no longer have the logic to show/hide the 2FA alert. That is now done in the `DirectDeposit` components, which also decide if if shows their child `BankInfo` components or not.